src/test/java/madalinapopescu/tests/tema5/demoqa/Links.java

```diff
@@ -0,0 +1,19 @@
+package madalinapopescu.tests.tema5.demoqa;
+
+import madalinapopescu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Links {
+    public static void main(String[] args) {
+        goLinks();
+    }
+    public static void goLinks() {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement links = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[6]"));
```

Comment:
Try to scroll first to the element before clicking it to avoid Selenium ElementClickInterceptedException. Thanks!